build.mjs

```diff
@@ -55,11 +57,154 @@ const filterOut = new Set([
 ]);
 
 const added = ["_malloc", "_free"];
+const typedefs = new Array();
+const funcs = new Array();
+const type_map = (qualType) => {
+  switch(qualType)
+  {
+    case "char **":                            return "string[]";  break;
```

Comment:
This could be `case "char **": return "LLVMStringRef";` right?

Reply:
its a double pointer, like string[]

Reply:
so maybe LLVMStringRef[]